decompiler/config/jak2/all-types.gc

```diff
@@ -1766,8 +1768,8 @@
 (define-extern *cos-poly-vec* vector)
 (define-extern vector-cos-rad! (function vector vector vector))
 (define-extern vector-sincos-rad! (function vector vector vector int))
-(define-extern sincos-rad! (function (pointer float) float int))
-(define-extern sincos! (function (pointer float) float int))
+(define-extern sincos-rad! (function vector float int))
+(define-extern sincos! (function vector float int))
```

Comment:
I'm trying to remember why I used `(pointer float)` here...
I think the function only writes to the first two values.